tests/L0/run_transformer/gpt_scaling_test.py

```diff
@@ -0,0 +1,85 @@
+import torch
+import torch.nn as nn
+import time
+import os
+import traceback
+import sys
+import copy
+import random
+import numpy as np
+import subprocess
+from apex.transformer.testing.commons import TEST_SUCCESS_MESSAGE
+import json
+
+def set_seed():
+	random.seed(0)
+	np.random.seed(0)
+	torch.manual_seed(0)
+	torch.cuda.manual_seed_all(0)

```

Comment:
[Question]
Doesn't this function need to call `apex.transformer.tensor_parallel.model_parallel_cuda_manual_seed(seed)`?
If this doesn't need to do so, could you tell me why?